cpp/hidapi/libusb/hid.c

```diff
@@ -369,7 +369,11 @@ static wchar_t *get_usb_string(libusb_device_handle *dev, uint8_t idx)
 	inbytes = len-2;
 	outptr = (char*) wbuf;
 	outbytes = sizeof(wbuf);
+#ifdef __NetBSD__
```

Comment:
The usual solution is to ifdef __NetBSD__ define ICONV_CONST const else define ICONV_CONST endif

Reply:
And putting the ifdef in some widely (preferably globally) included header file, then?

Reply:
It's usually used in a few places anyway. If this is just a single function it can be inlined in the function body.

Reply:
In this case, it's just this one use, in a source file that's been copied into this project from elsewhere. I guess if one were to do as you suggest, it would be either right next to the single use, or up near the top of hid.c, between the DEBUG_PRINTF definition and the FreeBSD DETACH_KERNEL_DRIVER stuff.

I'll keep it close, for now, I think.